### PR TITLE
[#420] Add browser player analytics

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

File: src/pages/project/project.jsx [hunk 1]
```diff
@@ -59,6 +60,8 @@ module.exports = React.createClass({
         return console.error('Could not fetch the page');
       }
 
+      platform.trackEvent('Browser Player', 'Open Project', 'Open Project Success', options.uri);
```
Comment (k88hudson) [inline src/pages/project/project.jsx]: `options.uri` is the url for the current page e.g. http://beta.webmaker.org/#/player?user=1&project=1

File: src/components/app-cta/app-cta.jsx [hunk 2]
```diff
@@ -29,6 +30,9 @@ var AppCta = React.createClass({
     });
   },
   openApp: function () {
+
+    platform.trackEvent('Browser Player', 'Click CTA', 'CTA: Open app', this.getDeepLink());
```
Comment (k88hudson) [inline src/components/app-cta/app-cta.jsx]: `getDeepLink` returns the deep link for the project in the app

**Comment by k88hudson** [comment]
@xmatthewx can you take a look at these events and let me know if they do what you were expecting?

**Comment by xmatthewx** [comment]
Looks good from where I sit, but I defer to @adamlofting. Context: user loads a project URL in browser & optionally taps "open app" button. Open app either loads the project in the app or loads the play store page.

File: src/components/app-cta/app-cta.jsx (hunk 2 above)
Comment (adamlofting) [inline src/components/app-cta/app-cta.jsx]: The fourth input in [this function](https://github.com/mozilla/webmaker-core/blob/develop/src/lib/platform.js#L167) ( `value` ) only accepts an integer in GA.

So I suggest we do this instead: 

``` js
platform.trackEvent('Browser Player', 'Click CTA Open App', this.getDeepLink());
```

File: src/pages/project/project.jsx (hunk 1 above)
Comment (adamlofting) [inline src/pages/project/project.jsx]: Same note as above:

``` js
platform.trackEvent('Browser Player', 'Open Project Success', options.uri);
```

**Comment by adamlofting** [comment]
R-

This just needs a small tweak to which values we pass into GA which I've noted in the comments. Then it's good to merge. :smile: 

I have one other question which isn't a blocker for this PR, but can we check that the page includes the main GA tracking tag somewhere as well? I've not looked at the codebase for the player yet, so this may already be in place.

**Comment by k88hudson** [comment]
@adamlofting oops sorry, this slipped through the cracks. thanks for looking it over, I've added changes as requested :+1:

**Comment by k88hudson** [comment]
@adamlofting also it does look as if the GA tracking tag is included here, does this look right? https://github.com/mozilla/webmaker-browser/blob/develop/static/index.html#L24

**Comment by adamlofting** [comment]
R+ for this PR.

I'm going to file another ticket to check if GA needs to be fully loaded before these events can be recorded. Because these events are being tracked on page load rather than triggered by a user interaction, it's possible we'll be firing them before GA is ready to listen. This was an issue on the old version of GA, but I'll investigate for Universal Analytics here.

**Comment by k88hudson** [comment]
@adamlofting ok, sounds good. Thanks!

**Pull request merged**